openhcl/page_pool_alloc/src/lib.rs

```diff
@@ -210,11 +106,13 @@ pub mod save_restore {
     #[mesh(package = "openhcl.pagepool")]
     pub struct PagePoolState {
         #[mesh(1)]
-        state: Vec<InnerState>,
+        state: Vec<SlotSavedState>,
         #[mesh(2)]
         device_ids: Vec<DeviceIdState>,
```

Comment:
If we're not using device IDs in the slots, why are we saving the device IDs?

Reply:
Is it useful at all to know if we've leaked one? if not eh we don't have to save them.

Reply:
eh just not saving them anymore.